src/core/server_message.h

```diff
@@ -46,13 +46,29 @@ class TritonServerMessage {
     msg.Write(&json_buffer_);
     base_ = json_buffer_.Base();
     byte_size_ = json_buffer_.Size();
+    from_json_ = true;
   }
 
   TritonServerMessage(std::string&& msg)
   {
     str_buffer_ = std::move(msg);
     base_ = str_buffer_.data();
     byte_size_ = str_buffer_.size();
+    from_json_ = false;
+  }
+
+  TritonServerMessage(const TritonServerMessage& rhs)
```

Comment:
Where are we copying a message?

Reply:
When instantiating the TritonBackend object https://github.com/NVIDIA/triton-inference-server/blob/master/src/backends/backend/triton_backend_manager.cc#L119

Reply:
OK, I guess it was copying incorrectly before...